### DISCUSSION TITLE
[ID 1723] Credential User Interface: Require trusted path for credential entry

### DISCUSSION THREAD

**Discussion by beerisgood** [0x6d69636b/windows_hardening#51] [General]
[This setting](https://github.com/0x6d69636b/windows_hardening/blob/master/lists/finding_list_0x6d69636b_machine.csv#L180) will break resetting Windows Hello PIN using FIDO2 hardware key after TPM change, right after Windows boots.

Last month i update my uEFI and with that, all uEFI settings were back to default which means no activated TPM.
As TPM is used by Windows Hello, the PIN wasn't usable - as expected.

No problem because of Hello reset feature, but it wasn't working because every time i press the needed Shift-Control-Del combination enforced by this policy, the window disappears.
The only way to get access was using normal login data.

In my opinion this feature isn't worth the risk for consumer.

**Comment by 0x6d69636b** [comment]
Okay, that's bad :( Doing a short research, I came across this article from Microsoft https://devblogs.microsoft.com/oldnewthing/20190312-00/?p=101092, and along with your experience; I'll remove the recommendation. No other guidelines recommend it anyway.

**Reply by beerisgood** [reply]
Thanks!
